editor/src/messages/portfolio/document/document_message_handler.rs

```diff
@@ -1056,36 +1040,39 @@ impl DocumentMessageHandler {
 		};
 
 		// Find the primary input type of the node graph
-		let primary_input_type = node_network.input_types().next().clone();
-		let response = match primary_input_type {
-			// Only calclate the frame if the primary input is an image
-			Some(ty) if ty == concrete!(ImageFrame<Color>) => {
-				// Calculate the size of the region to be exported
-				let old_transforms = self.remove_document_transform();
-				let transform = self.document_legacy.multiply_transforms(&layer_path).unwrap();
-				let size = DVec2::new(transform.transform_vector2(DVec2::new(1., 0.)).length(), transform.transform_vector2(DVec2::new(0., 1.)).length());
+		let input_frame = node_network.nodes.iter().find(|(_, node)| node.name == "Input Frame");
```

Comment:
Ah true, I forgot about that. I guess this will only really get resolved once we transition to having a screenshot node. Do you mind adding a TODO comment explaining that?

Reply:
*(just a note: we may or may not end up refactoring the screenshot node, or we might just jump straight into full nodes, depending on timing*)

Reply:
the screenshot node would be the async fully node version. (You could also call it rasterize if you want)

Reply:
Yeah, I'm just talking about how we might end up being very close to the full node system which would obviate the screenshot node entirely. The screenshot node (regardless of its name) is just a stopgap until then.